### ISSUE TITLE
GUI : FED : Local user management: Modals

### ISSUE THREAD

**Issue by derick-montague** [ibm-openbmc/dev#1723]
## Overview
The use of the bootstrap vue form group component does not meet accessibility guidelines. It also does not render consistent markup that can be used to style the form labels to meet the design style guide.

### Accessibility violations
This component adds the `role` attribute to `form-group` wrapper. With this attribute set on the element there also needs to be either an `aria-label` or `aria-labelledby` attribute.

#### Violation info
- https://aat.w3ibm.mybluemix.net/token/82c6f45f-caf9-43f4-b4ad-8b6f891c89fb/01b78da0-e8a7-4010-b663-ba8cf47be32d/latest///doc/w3/help/en-US/idhi_accessibility_check_g1169.html
- https://www.ibm.com/able/guidelines/ci162/bypass_blocks.html
- https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks

#### Inconsistency info
The lack of consistent CSS selectors results in inconsistencies in the display of the form section labels.

**Comment by yoshiemuranaka** [comment]
Resolved by https://gerrit.openbmc-project.xyz/c/openbmc/webui-vue/+/30141
